prac_06/guitar.py

```diff
@@ -0,0 +1,27 @@
+"""
+Guitar Class file
+Estimate: 30 minutes
+Actual:   35 minutes
+"""
+CURRENT_YEAR = 2022
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """basic class for guitar information"""
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name} ({self.year}) : ${self.cost:.2f}"
```

Comment:
Doesn't format thousands correctly with a comma. I found adding a comma before the full stop works
eg. {self.cost:,.2f}